README.md

```diff
@@ -261,6 +200,44 @@ Array<{
     address: string,
     amount: string,
   }>,
+  withdrawals: Array<{ address: string,
+    amount: string
+  }>,
+  certificates: Array<{|
+    king: 'StakeRegistration',
+    stakeCredential: string,
+  |} | {|
+    king: 'StakeDeregistration',
+    stakeCredential: string,
+  |} | {|
+    king: 'StakeDelegation',
+    stakeCredential: string,
```

Comment:
kind of strange how just these fields randomly are camelCase but everywhere else in this library is underscore

Reply:
you're right.  I did a sloppy job on the readme. :-/